dango/testing/tests/vesting.rs

```diff
@@ -0,0 +1,554 @@
+use {
+    dango_testing::{setup_test_naive, Accounts, TestSuite},
+    dango_types::vesting::{self, QueryPositionRequest, Schedule, VestingStatus},
+    grug::{
+        Addr, Addressable, Coin, Coins, Duration, Inner, MultiplyFraction, ResultExt, Timestamp,
+        Udec128, Uint128,
+    },
+    grug_app::NaiveProposalPreparer,
+    std::sync::LazyLock,
+};
+
+static TEST_AMOUNT: LazyLock<Coin> = LazyLock::new(|| Coin::new("uusdc", 100).unwrap());
+
+const ONE_MONTH: Duration = Duration::from_weeks(4);
+const ONE_DAY: Duration = Duration::from_days(1);
+
+fn setup_test() -> (TestSuite<NaiveProposalPreparer>, Accounts, Addr) {
+    let (suite, accounts, _codes, contracts) = setup_test_naive();
+
+    (suite, accounts, contracts.vesting)
+}
+
+#[test]
+fn missing_funds() {
+    let (mut suite, mut accounts, vesting_addr) = setup_test();
+
+    suite
+        .execute(
+            &mut accounts.owner,
+            vesting_addr,
+            &vesting::ExecuteMsg::CreatePosition {
+                user: accounts.relayer.address(),
+                schedule: Schedule {
+                    start_time: Duration::from_seconds(0),
+                    cliff: Duration::from_seconds(0),
+                    vesting: Duration::from_seconds(0),
+                },
+            },
+            Coins::default(),
+        )
+        .should_fail_with_error("invalid payment: expecting 1 coins, found 0");
```

Comment:
Instead of putting a string here. we should use

```rust
.should_fail_with_error(StdError::invalid_payment(1, 0));
```

This way, if we change the error message, the tests would still pass.

Reply:
This problem is not limited to vesting, so let's fix them all in a separate PR...